Sources/Async/EventLoop/EventLoop.swift

```diff
@@ -28,6 +36,10 @@ public protocol EventLoop: Worker {
     /// is ready to write data and the event source is resumed.
     func onWritable(descriptor: Int32, _ callback: @escaping EventCallback) -> EventSource
 
+    /// Creates a new timer event source.
+    /// This callback will be called perodically when the timeout is reached.
+    func onTimeout(timeout: Int, _ callback: @escaping EventLoop.EventCallback) -> EventSource
```

Comment:
What is the `timout: Int`? Is it seconds, MS or Nanoseconds even?

Reply:
milliseconds. I will change the naming here to be more clear!